package.json

```diff
@@ -8,25 +8,17 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "grunt": "^0.4.5",
-    "grunt-contrib-jshint": "~0.10.0",
-    "grunt-contrib-nodeunit": "~0.4.1",
-    "grunt-istanbul": "^0.6.1",
-    "grunt-mocha": "^0.4.13",
-    "grunt-mocha-test": "^0.12.7",
     "mocha": "^2.2.5",
     "node-pty": "^0.9.0"
   },
   "dependencies": {
     "async": "^2.1.1",
     "bee-queue": "^1.2.2",
     "bluebird": "^3.5.5",
-    "body-parser": "1.9.x",
```

Comment:
in use

Reply:
This doesn't appear to be in use outside the `api` directory, which has its own `package.json` file:

https://github.com/firewalla/firewalla/blob/4550b0756ce72d253290d437c5f7d54a8d213005/api/package.json#L9